### DISCUSSION TITLE
I have a very serious problem with web3Modal/react.

### DISCUSSION THREAD

**Discussion by SobeitB** [WalletConnect/walletconnect-monorepo#2282] [Announcements]
Hi all, I have a very serious problem with web3Modal/react. 
1. it's normal when you call modal via connect({connector:connectorWC}) // from wagmi and modal window gives too small number of available wallets (no metamask and trust wallet at least). And when connecting on the phone through the browser, trust and other wallets don't connect at all!
2. when i use open from useWeb3Modal hook from @web3modal/react library modal window is called on page relaunch, even though i don't use open() function.

Please help.
my code: 
![image](https://user-images.githubusercontent.com/91097143/235480108-730007b5-eebe-44f8-adee-2887332fcdc6.png)
![image](https://user-images.githubusercontent.com/91097143/235480135-5f9bba1c-b8b3-4858-8e3e-0d349f06169b.png)
![image](https://user-images.githubusercontent.com/91097143/235480173-c0347551-a752-4b43-85a6-2181b0d207d5.png)
// I commented out Web3Modal, because it causes a modal window to open on page relaunch
![image](https://user-images.githubusercontent.com/91097143/235480312-80114699-16a3-422e-9b73-b612e45f4f84.png)

**Comment by glitch-txs** [comment]
Wagmi's WalletConnect connector works with Standalone modal which does not include extension wallets like MetaMask. 

@web3modal/react is not from wagmi but from Web3Modal SDK library, so there are some mixed dependencies here. 

If you want to use WalletConnect with wallet extensions like MetaMask you should use [Web3Modal SDK](https://docs.walletconnect.com/2.0/web3modal/react/installation) instead of wagmi connectors.
